tests/e2e/vsan_stretched_cluster.go

```
@@ -2473,4 +2509,291 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
+		}
+
+		for i := volumeOpsScale / 2; i < volumeOpsScale; i++ {
+			pvcs = append(pvcs, pvclaims[i])
+		}
+
+		// Get the list of csi pods running in CSI namespace
+		csipods, err := client.CoreV1().Pods(csiNs).List(ctx, metav1.ListOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Bring down the secondary site while deleting pv")
+		var wg sync.WaitGroup
+		wg.Add(3)
+		go deletePvcInParallel(client, pvcs, namespace, &wg)
+		go deletePvInParallel(client, pvs, &wg)
+		go siteFailureInParallel(false, &wg)
+		wg.Wait()
+
+		defer func() {
+			ginkgo.By("Bring up the secondary site before terminating the test")
+			if len(fds.hostsDown) > 0 {
+				siteRestore(false)
+				fds.hostsDown = []string{}
+			}
+		}()
+
+		ginkgo.By("Wait for k8s cluster to be healthy")
+		wait4AllK8sNodesToBeUp(ctx, client, nodeList)
+		err = waitForAllNodes2BeReady(ctx, client)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		// Check if csi pods are running fine after site failure
+		err = fpod.WaitForPodsRunningReady(client, csiNs, int32(csipods.Size()), 0, pollTimeout, nil)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify all PVCs are deleted from namespace")
+		for _, pvc := range pvclaims {
+			err = waitForPvcToBeDeleted(ctx, client, pvc.Name, namespace)
+		}
+
+		ginkgo.By("Verify PVs, volumes are deleted from CNS")
+		for _, pv := range pvs {
+			err := fpv.WaitForPersistentVolumeDeleted(client, pv.Name, poll,
+				pollTimeout)
+			framework.Logf("Persistent Volume %v still not deleted with err %v", pv.Name, err)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			volumeHandle := pv.Spec.CSI.VolumeHandle
+			// Orphan volumes may be left over here, hence logging those PVs and ignoring the error for now.
+			err = e2eVSphere.waitForCNSVolumeToBeDeleted(volumeHandle)
+			framework.Logf("Volume %v still not deleted from CNS with err %v", pv.Name, err)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

you mentioned above that we are ignoring the error, so remove this